docs/source/functional_sox_compatibility.rst

```diff
@@ -0,0 +1,24 @@
+.. role:: hidden
+    :class: hidden-section
+
+torchaudio.functional_sox_compatibility
```

Comment:
For now, let's actually move all those functionals into the functional. We should only claim compatibility if we test agreement between sox and pytorch. We may revisit that decision at a later time.